roles/openshift_health_checker/openshift_checks/docker_image_availability.py

```diff
@@ -0,0 +1,76 @@
+# pylint: disable=missing-docstring
+from openshift_checks import OpenShiftCheck, get_var
+from openshift_checks.mixins import NotContainerizedMixin
+
+
+class DockerImageAvailability(NotContainerizedMixin, OpenShiftCheck):
```

Comment:
Agreed, and actually it would influence which images are required. For a containerized install you would want the `openshift/origin` or `openshift3/ose` image to be available in addition to pod/registry/router/etc.